exotations/exotica_core_task_maps/exotica_plugins.xml

```diff
@@ -71,4 +72,7 @@
   <class name="exotica/GazeAtConstraint" type="exotica::GazeAtConstraint" base_class_type="exotica::TaskMap">
     <description>Keeps a gaze position within a cone in the end-effector frame defined by a given viewing angle. This task map can only be used as an inequality constraint.</description>
   </class>
+  <class name="exotica/EffBoxConstraint" type="exotica::EffBoxConstraint" base_class_type="exotica::TaskMap">
+    <description>Limits the end-effector motion to a box in some reference frame.</description>
```

Comment:
Right now, this is an inequality constraint (not explicitly stated). With an initialiser switch, it could also be used as a cost.

Reply:
Addressed in latest commit. If possible, re initializer switch, I suggest making this a TODO for a later date. Also, given this future functionality, perhaps it makes sense to change the task map name from `EffBoxConstraint` to `EffBox`?

Reply:
@wxmerkt shall i go ahead with name change and add the todo?